resources/styles/global/tutor-bootstrap.less

```diff
@@ -0,0 +1,50 @@
+// Core variables and mixins
+@import "@{bootstrap-path}variables";
```

Comment:
My only comment is it looks "weird" when there is no `/` between the `@{bootstrap-path}` and `variables`. Having 2 `/` is a safer fallback than having none (if the user forgot to add the trailing `/` to the variable definition)